include/cantera/numerics/CVodesIntegrator.h

```diff
@@ -35,7 +35,6 @@ class CVodesIntegrator : public Integrator
     virtual void setTolerances(double reltol, size_t n, double* abstol);
     virtual void setTolerances(double reltol, double abstol);
     virtual void setSensitivityTolerances(double reltol, double abstol);
-    virtual void setProblemType(int probtype);
```

Comment:
To rename this function `setLinearSolverType`, this function needs to be marked as deprecated and issue a deprecation warning (using the `warn_deprecated` function), rather than simply removing it without notice.